### PR TITLE
[16.0][ADD] rma_procurement_customer

### PR DESCRIPTION
**Pull request by sbejaoui**
This module extends the `stock_procurement_customer` module to ensure that the customer information is automatically propagated to any stock pickings created as part of an RMA process.

### CODE REVIEW

**Comment by sbejaoui** [comment]
cc/ @lmignon @rousseldenis

**Review by lmignon — APPROVED** [review]
LGTM. Thank you for the glue module.

File: rma_procurement_customer/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This module extends the `rma_procurement_customer` module to ensure that the
```
Comment (chienandalu) [inline rma_procurement_customer/readme/DESCRIPTION.md]: Maybe you mean:

```suggestion
This module extends the `stock_procurement_customer` module to ensure that the
```

**Review by rousseldenis — APPROVED** [review]
LGTM

**Review by chienandalu — APPROVED** [review]

**Pull request merged**